pkgs/by-name/uk/ukmm/package.nix

```
@@ -70,6 +73,23 @@ rustPlatform.buildRustPackage (finalAttrs: {
 
   passthru.updateScript = nix-update-script { };
 
+  postInstall = ''
+    install -Dm444 assets/ukmm.png  $out/share/icons/hicolor/256x256/apps/ukmm.png
+  '';
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "Breath of the Wild Mod Manager";
+      exec = "ukmm";
+      icon = "ukmm";
+      desktopName = "UKMM";
+      categories = [
+        "Game"
```

I would suggest adding

```suggestion
        "Game"
        "Utility"
```

or only "Utility" as it's a tool rather than the game itself. But if it launches the game too, "Game" should also be okay

See https://specifications.freedesktop.org/menu-spec/0.92/category-registry.html

---

It also looks like ukmm will install a desktop file to `~/.local/share/applications/ukmm.desktop` with Settings > General > Register 1-Click Handler.

I think our desktop file should match the behaviour of the generated one, specifically by handling links from Game Banana:
```nix
exec = "ukmm %u";
mimeTypes = [ "x-scheme-handler/bcml" ];
```

https://github.com/NiceneNerd/UKMM/blob/90678f93455f1bf6f0a22bf5704c852d942d59a1/src/gui/tasks/handlers.rs#L32-L41